docs/ref/contrib/staticfiles.txt

```diff
@@ -128,6 +128,25 @@ For a full list of options, refer to the commands own help by running::
 
    $ python manage.py collectstatic --help
 
+Customizing the ignored pattern list
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The default ignored pattern list, ``['CVS', '.*', '*~']``, can be customized on
+a more stable way than providing the ``--ignore`` command option at each
+``collectstatic`` invocation. Provide a custom :class:`~django.apps.AppConfig`
+class, override the ``collectstatic_ignore_patterns`` attribute of this class
+and specify that class path inside your :setting:`INSTALLED_APPS` setting::
+
```

Comment:
Add `.. code-block:: python` as we're in a `.. highlight:: console` context, so highlighting doesn't work correctly otherwise.